package.json

```diff
@@ -13,72 +13,64 @@
     "build:copy": "find src -type f \\( -name \"*.yml\" -or -name \"*.json\" -or -name \"*.xlsx\" \\) | cpio -pdm .build && find tests -type f \\( -name \"*.yml\" -or -name \"*.json\" \\)  | cpio -pdm .build",
     "build:docker": "docker-compose -f docker-compose.yml up -d --build",
     "test": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.unitTest.ts\" --coverage",
-    "security-checks": "git secrets --scan && git log -p | scanrepo",
-    "tslint": "tslint --project tslint.json",
+    "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand --passWithNoTests",
+    "tslint": "tslint src/**/*.ts tests/**/*.ts",
     "prepush": "npm test && npm run tslint",
+    "security-checks": "git secrets --scan && git log -p | scanrepo",
     "sonar-scanner": "BRANCH=local npm run tslint && npm run coverage && sonar-scanner",
     "coverage": "jest --testMatch=\"**/*.*Test.ts\" --coverage",
     "audit-dep": "npm audit"
   },
-  "nyc": {
-    "include": [
-      "src/**/*.ts",
-      "src/**/*.tsx"
-    ],
-    "extension": [
-      ".ts",
-      ".tsx"
-    ],
-    "require": [
-      "ts-node/register"
-    ],
-    "reporter": [
-      "text-summary",
-      "html"
-    ],
-    "sourceMap": true,
-    "instrument": true
-  },
   "author": "",
   "license": "ISC",
   "dependencies": {
     "aws-lambda": "^0.1.2",
-    "aws-sdk": "^2.392.0",
-    "aws-xray-sdk": "^2.3.3",
-    "exceljs": "^1.7.0",
+    "aws-sdk": "^2.560.0",
+    "aws-xray-sdk": "^2.4.0",
+    "exceljs": "1.7.0",
     "moment": "^2.24.0",
-    "moment-timezone": "^0.5.23",
+    "moment-timezone": "^0.5.27",
     "node-yaml": "^3.2.0",
-    "reflect-metadata": "^0.1.13",
-    "notifications-node-client": "^4.6.0"
+    "notifications-node-client": "^4.6.0",
+    "reflect-metadata": "^0.1.13"
   },
   "devDependencies": {
-    "@types/aws-lambda": "^8.10.17",
-    "@types/chai": "^4.1.7",
-    "@types/jest": "^24.0.15",
+    "@types/aws-lambda": "^8.10.34",
+    "@types/chai": "^4.2.4",
+    "@types/jest": "^24.0.21",
     "@types/jest-plugin-context": "^2.9.2",
     "@types/moment-timezone": "^0.5.12",
-    "@types/node": "^10.12.18",
-    "@types/request-promise": "^4.1.42",
-    "@types/sinon": "^7.0.13",
+    "@types/node": "^12.12.3",
+    "@types/request-promise": "^4.1.44",
+    "@types/sinon": "^7.5.0",
     "aws-lambda-mock-context": "^3.2.1",
     "aws-sdk-mock": "^4.5.0",
     "chai": "^4.2.0",
-    "gulp": "^4.0.0",
+    "gulp": "^4.0.2",
     "husky": "^3.0.9",
     "jest": "^24.8.0",
     "jest-plugin-context": "^2.9.0",
-    "nyc": "^14.1.1",
-    "serverless": "^1.42.3",
+    "serverless": "^1.56.1",
     "serverless-dependency-invoke": "^0.0.9",
```

Comment:
I'd suggest to also remove the dependency here

Reply:
It is - this is the "before" side.

Reply:
Sorry, I wasn't explicit enough. I saw that you removed the serverless-dependency-invoke config so I was suggesting to remove `"serverless-dependency-invoke": "^0.0.9"` also.